samples/client/petstore-security-test/scala-httpclient/.gitignore

```diff
@@ -15,3 +16,7 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+
+# IntelliJ specific
+.idea
+*.iml
```

Comment:
`*.iml` should not be ignored. See reasoning: https://github.com/joeblau/gitignore.io/issues/186#issuecomment-215987721